src/main/resources/explanations/challenge1_hint.adoc

```diff
@@ -2,10 +2,18 @@ As the text of the challenge says, we are looking for a secret called `password`
 
 You can solve this challenge by the following steps:
 
+1. When you have the source code available you can find hardcoded string simply by searching through the code. grep can be used to help this process as it searches for strings or patterns of strings in large blocks of text. Use the below steps to find the answer
+- Access a terminal with https://man7.org/linux/man-pages/man1/grep.1.html[*grep*] and https://git-scm.com/[*git*] installed
+- Clone the repo with `git clone https://github.com/commjoen/wrongsecrets`.
+- Navigate to the Java code where the check for the right answer happens `cd src/main/java/org/owasp/wrongsecrets/`.
+- Use grep recursively to look for the password string `grep -r password`.
 
-1. Use https://github.com/trufflesecurity/truffleHog[*Trufflehog*] to scan the repository for secrets:
-- Read up on the instructions of the tool at https://github.com/trufflesecurity/truffleHog[Trufflehog] and install it using `pip3 install truffleHog` or use its docker container.
-- Now run Trufflehog at our repository: `trufflehog https://github.com/commjoen/wrongsecrets`. Can you find the value of `public static String password` in the output?
-2. When you do not have the source code available, try to obtain the actual application to reverse engineer it. The application, in this case, is a Jar file, which is a set of java class files together with a few resources.
+2. An automated tool like https://github.com/awslabs/git-secrets[*Git-secrets*] can often help out. In this case it needs a little set up.
+- Clone the repo with `git clone https://github.com/commjoen/wrongsecrets`.
+- Follow the instructions https://github.com/awslabs/git-secrets[here] to install Git leaks.
```

Comment:
Ive been using Gitleaks too much recently and its stuck in my head. I'll have to change this to Git Secrets